### PR TITLE
Bug Fixes: 0916246, 916243

### PR DESCRIPTION
**Pull request by cs-ankit-gupta**
Description
1. Changed Tactic and Group from Lookup field to Many To Many in Alert and Incident.
2. Correlated Tactic and Group with Alert and Incident within all the playbooks.
3. Removed Mitre Admin Role.
4. Removed Hunt permission from Full App Permission.
5. Reorganized the MITRE module in navigation.

### CODE REVIEW

File: modules/alerts/mmd.json
Comment (cs-rohit-khune) [inline modules/alerts/mmd.json:1]: Remove mmd changes for alert and incident mmd and test deploying SP. If changes do not reflect for alert and incident then keep these files

File: modules/incidents/mmd.json
Comment (cs-rohit-khune) [inline modules/incidents/mmd.json:1]: Remove mmd changes for alert and incident mmd and test deploying SP. If changes do not reflect for alert and incident then keep these files

**Review by cs-rohit-khune — CHANGES_REQUESTED** [review]

File: modules/alerts/mmd.json
Comment (cs-ankit-gupta) [reply modules/alerts/mmd.json:1]: If we reomove mmd changes of alert, then in fresh install it work properly.
In update "Group" and "Tactic" field type didn't get update.

File: modules/incidents/mmd.json
Comment (cs-ankit-gupta) [reply modules/incidents/mmd.json:1]: If we reomove mmd changes of Incidnet, then in fresh install it work properly.
In update "Group" and "Tactic" field type didn't get update.

**Review by cs-rohit-khune — APPROVED** [review]
Approved

**Pull request merged**